### PR TITLE
enable reattachments of zero value txs

### PR DESCRIPTION
**Pull request by chrisdukakis**

### CODE REVIEW

File: ui/js/ui.history.js [hunk 1]
```diff
@@ -64,6 +59,27 @@ var UI = (function(UI, $, undefined) {
...
+              iota.api.getLatestInclusion(txs.filter(tx => tx.currentIndex === 0).map(tx => tx.hash), function (err, states) {
+                if (err) {
+                  return
+                }
+                isReattachable = states.indexOf(true) === -1
+                renderBundleModal(isReattachable)
+              })
+            })
+          }
+          renderBundleModal(isReattachable)
```
Comment (th0br0) [inline ui/js/ui.history.js]: this is missing an else I believe.

Comment (chrisdukakis) [reply ui/js/ui.history.js]: it returns above inside the if block (L69). I actually think of this often, what's the best coding style...

Comment (th0br0) [reply ui/js/ui.history.js]: yes. but if `isReattachable` is true already then you don't need to call `findTransactions`!?

Comment (th0br0) [reply ui/js/ui.history.js]: nevermind. too late here, you're right.

**Comment by chrisdukakis** [comment]
#779 includes this feature

**Pull request closed without merging**